engine_wrapper.py

```diff
@@ -29,6 +30,12 @@
 
 @contextmanager
 def create_engine(engine_config: config.Configuration) -> Generator[EngineWrapper, None, None]:
+    """
+    Create the engine.
```

Comment:
Add: Use in a with-block to automatically close the engine when exiting the game.